src/frontend/cypress/utility/auth.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
```

Comment:
this looks very similar to the command added to cypress in support/commands. What's the difference/why is there dupilication?